bHY6yh-d18923/cod-yyyiou_bHY6yh-d18923.xml

```diff
@@ -0,0 +1,426 @@
+          guidelines for a diplomatic edition.</p>
+      </editorialDecl>
+    </encodingDesc>
+    <revisionDesc status="draft">
+      <listChange>
+        <change when="2023-07-17" status="draft" n="0.0.0">
+          <p>Created file for the first time.</p>
+        </change>
+      </listChange>
+    </revisionDesc>
+  </teiHeader>
+  <text xml:lang="la">
+    <front>
+      <div xml:id="starts-on">
+        <pb ed="#C" n="27-r"/>
+        <cb ed="#C" n="b"/>
+      </div>
+    </front>
+    <body>
+      <div xml:id="bHY6yh-d18923">
+        <head xml:id="bHY6yh-d18923-Hd1e108">Rubrica 2</head>
+        <head xml:id="bHY6yh-d18923-Hd1e111" type="question-title">De bonitate et prauitate</head>
+        <p xml:id="bHY6yh-d18923-d1e114">
+          De prauitate et ipsu
+          <lb ed="#C"/>iutate capitulum
+        </p>
+        <!--the next paragraph begins on the same line as the previous paragraph-->
+        <p xml:id="bHY6yh-d18923-d1e119">
+          <ref xml:id="bHY6yh-d18923-Rd1e116">IN genesi
+          <lb ed="#C"/>VIII capitulo</ref> <quote xml:id="bHY6yh-d18923-Qd1e123">Sensus et cogitatio humani cordis in malum
+          <lb ed="#C"/>poena sunt ab adolescentia sua</quote>
+        </p>
+        <p xml:id="bHY6yh-d18923-d1e133">
+          ¶ <ref xml:id="bHY6yh-d18923-Rd1e132">Psalmista psalmo IX</ref>
+          <lb ed="#C"/><quote xml:id="bHY6yh-d18923-Qd1e137">Cuius maledictione os plenum est et amaritu<lb ed="#C" break="no"/>dine 
+          et dolo sublingua eius labor et dolor</quote>
+        </p>
+        <p xml:id="bHY6yh-d18923-d1e146">
+          <lb ed="#C"/>¶ <ref xml:id="bHY6yh-d18923-Rd1e145">Ibidem psalmus XIII</ref> <quote xml:id="bHY6yh-d18923-Qd1e148">Quorum maledictione
+          <lb ed="#C"/>et amaritudine plenum est ueloces pedes e<lb ed="#C" break="no"/>orum 
+          ad effundendum sanguinem</quote>
+        </p>
+        <p xml:id="bHY6yh-d18923-d1e158">
+          ¶ <ref xml:id="bHY6yh-d18923-Rd1e159">Ibidem
+          <lb ed="#C"/>XXV</ref> <quote xml:id="bHY6yh-d18923-Qd1e160">In quorum manibus iniquitates sunt
+          <lb ed="#C"/>dextera eorum repleta est muneribus</quote>
+        </p>
+        <p xml:id="bHY6yh-d18923-d1e168">
+          ¶ <ref xml:id="bHY6yh-d18923-Rd1e171">Idem
+          <lb ed="#C"/>34</ref> <quote xml:id="bHY6yh-d18923-Qd1e170">quis est qui uult uitam diligit dies
+          <lb ed="#C"/>uidere bonos prohibeat linguam suam a ma<lb ed="#C" break="no"/>lo 
+          et labia sua ne loquantur dolum diuerte
+          <lb ed="#C"/>a malo et fac bonum inquire pacem et per
+          <lb ed="#C"/>sequere eam</quote>
+        </p>
+        <p xml:id="bHY6yh-d18923-d1e182">
+          ¶ <ref xml:id="bHY6yh-d18923-Rd1e187">Idem 36</ref> <quote xml:id="bHY6yh-d18923-Qd1e184">Considerat pec<lb ed="#C" break="no"/>cator 
+          iustum et querit mortificare eum dominus
+          <lb ed="#C"/>autem non derelinquet eum in manibus sius</quote>
```

Comment:
"sius" isn't a latin word. If it isn't a Latin word, I would review the transcription a second time. Only after double checking we I leave obviously wrong word as is, but it should also be surrounded by `<sic>`. In this case the word is definitely "suis"

```suggestion
          <lb ed="#C"/>autem non derelinquet eum in manibus suis</quote>
```